internal/controller/authenticator/webhookcachefiller/webhookcachefiller.go

```diff
@@ -136,10 +242,201 @@ func newWebhookAuthenticator(
 	//  then use client.JSONConfig as clientConfig
 	clientConfig, err := webhookutil.LoadKubeconfig(temp.Name(), customDial)
 	if err != nil {
-		return nil, err
+		errText := "unable to load kubeconfig"
```

Comment:
Maybe add a comment that this block is not covered by unit tests? That would tell the next person who edits this file to be extra careful here.

Reply:
👍